requirements.txt

```diff
@@ -1,7 +1,7 @@
 pip >= 7.0.0
 pytest
 mock
-IPython >= 1.2.1
+IPython == 4.0.0
```

Comment:
This is managed via https://github.com/log2timeline/l2tpreg/blob/master/dependencies.ini and https://github.com/log2timeline/l2tpreg/blob/master/utils/update_dependencies.py to update all files with dependency versions.